Zoo/Workers/Cleaner.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Zoo.Animals;
+
+namespace Zoo.Workers
+{
+    public class Cleaner : Worker
+    {
+        public Cleaner(string name, Area WorkArea) : base(name)
+        {
+            this.WorkArea = WorkArea;
+            this.WorkTime = 2;
```

Comment:
Since no DMs are declared in this class all of them should be sent to base class.
(Relevent to all workers and animals).